scripts/parseJunit.js

```diff
@@ -0,0 +1,27 @@
+const Parser = require("junitxml-to-javascript");
+const { notifyFail, notifyPass } = require("./notifySlack");
+
+const passed = (suite) => suite.succeeded === suite.tests;
+
+/**
+ * Determine how many test files passed and call notify functions
+ */
+const parseAllFiles = async () => {
+  const fileName = __dirname + "/../reports/junit/junit.xml";
```

Comment:
[Here's an example of an xml file that comes out of a test run.](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5dd94a86aab41e1a88900f15/605ceff9c02c560885e552af-0-build/artifacts/reports/junit/junit.xml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210325T204120Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210325%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=f8c29220980265ab2bc667991633248f4deb1d911af3c99ac344559084fadf39)

The parser does not do much except turn this into a JavaScript object that has a `testsuites` key whose value is an array of objects with properties like `name`, `succeeded` (number of test cases that passed), and `tests` (number of test cases).